websocket.go

```diff
@@ -34,4 +47,15 @@ func wsHandler(hub *Hub, w http.ResponseWriter, r *http.Request) {
 	//register client
 	hub.clients[conn] = true
 	log.Println(hub.clients)
+
+	err = conn.WriteMessage(1, []byte("Hi Client!"))
```

Comment:
use valid json instead of raw string msg;
also for loops that read/write into conn should be invoked in goroutines;